libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1453,6 +1453,27 @@ void AP_Arming::Log_Write_Disarm(const AP_Arming::Method method)
     };
     AP::logger().WriteCriticalBlock(&pkt, sizeof(pkt));
     AP::logger().Write_Event(LogEvent::DISARMED);
+
+    if (method == AP_Arming::Method::TERMINATION ||
```

Comment:
If you use a switch in here instead then when someone adds a new method they are forced to evaluate whether it should be added here.